src/main/java/duke/Task.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+/**
+ * Encapsulates the abstract Task class.
+ */
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public abstract String getTypeIndicator();
```

Comment:
Since all task has a type, would it be better to have a type variable in the abstract class?

Reply:
Not sure why my other comments didn't show up. Other things I commented on were 
1. Some missing header comments
2. Since all task has an isExit, would it be better to include it as a variable in the abstract class?
3. Very clear and readable method names. Nice!

Reply:
Thanks for your input! Regarding (2), I believe you are referring to the command classes. Unfortunately I'm unable to implement the suggestion since I made ```Command``` an interface